engines/bastion_katello/app/assets/javascripts/bastion_katello/ostree-branches/details/ostree-branches-details-repositories.controller.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.ostree-branches.controller:OstreeBranchesDetailsRepositoriesController
+ *
+ * @requires $scope
+ * @requires $q
+ * @requires Nutupane
+ * @requires Repository
+ * @requires Environment
+ * @requires ContentView
+ * @requires CurrentOrganization
+ *
+ * @description
+ *   Provides the functionality for the errata details repositories page.
+ */
+angular.module('Bastion.ostree-branches').controller('OstreeBranchesDetailsRepositoriesController',
+    ['$scope', '$q', 'Nutupane', 'Repository', 'Environment', 'ContentView', 'CurrentOrganization',
+    function ($scope, $q, Nutupane, Repository, Environment, ContentView, CurrentOrganization) {
+        var repositoriesNutupane, environment, contentView;
+        var params = {
```

Comment:
All vars should be declared on one line.  I'm surprised this wasn't caught by our style checking @ehelms.